hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/NSSummaryEndpoint.java

```diff
@@ -62,6 +70,129 @@ public NSSummaryEndpoint(ReconNamespaceSummaryManager namespaceSummaryManager,
     this.reconSCM = reconSCM;
   }
 
+  @GET
+  @Path("/entitymetrics")
+  public Response getEntityMetrics(@QueryParam("path") String path,
+                                   @DefaultValue("size")
```

Comment:
orderby should include "age" also a filter and as default

Reply:
@sumitagrawl - Thanks for reviewing. It has been fixed. Pls re-review.